File: front/src/models/imc.ts
import { Aluno } from "./aluno";

export interface Imc {
    imcId?: number;
    peso: number;
    altura: number;
    alunoId: number;
    imcConta: number;
    aluno?: Aluno;
    criadoEm?: string;
    classificacao: string;
}

File: API/API/Models/Imc.cs
namespace API.Models;

public class Imc{
    public int ImcId { get; set; }
    public double ImcConta { get; set; }
    public double Altura { get; set; }
    public double Peso { get; set; }
    public string Classificacao { get; set; }
    public DateTime CriadoEm { get; set; } = DateTime.Now;
    public int AlunoId { get; set; }
    public Aluno? Aluno { get; set; }
}